Assets/Scripts/Gamestate/Round.cs

```diff
@@ -119,7 +119,7 @@ private void CheckWinCondition(PlayerManager lastKiller)
     {
         if (livingPlayers.Count < 2)
         {
-            winner = livingPlayers.FirstOrDefault()?.identity;
+            winner = lastKiller?.identity;
```

Comment:
Last living player is not necessarily last killer, is this change intended to change the logic or was it a refactor with unintended consequences?

Reply:
Was a refactor, the method's parameter was unused and this change technically resolved that. Reverted and removed the parameter.